packages/apollo-server-client/index.js

```diff
@@ -0,0 +1,242 @@
+const { ApolloClient, InMemoryCache, ApolloLink } = require('@apollo/client')
+const { BatchHttpLink } = require('@apollo/client/link/batch-http')
+const { createUploadLink } = require('apollo-upload-client')
+const { onError }  = require('apollo-link-error')
+const { RetryLink } = require('@apollo/client/link/retry')
+const { getLogger } = require('@condo/keystone/logging')
+const fetch = require('cross-fetch/polyfill').fetch
+const { chunk: splitArray } = require('lodash')
+
+const { MAX_REQUESTS_IN_BATCH, MAX_MODIFY_OPERATIONS_IN_REQUEST, MAX_RETRIES_ON_NETWORK_ERROR, LOAD_CHUNK_SIZE } = require('./constants')
+const { SIGNIN_BY_EMAIL_MUTATION, SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION } = require('./lib/gql')
+
+class ApolloServerClient {
+
+    client
+    clientName
+    authToken
+    userId
+    locale
+    endpoint
+    authRequisites = {}
+    logger
+
+    /**
+     *
+     * @param endpoint - https://condo.d.doma.ai/admin/api
+     * @param authRequisites - can be { identity: 'service-user@doma.ai', secret: 'password' } or { phone: '+7911....', password: '' }
+     * @param clientName - logger name
+     * @param locale - for server side translated texts
+     */
+    constructor (endpoint, authRequisites = {}, { clientName = 'apollo-server-client', locale = 'ru' } = {}) {
```

Comment:
What about to make setting `clientName` obligatory, without default value.
This way there is more likely that developers will choose unique name for modules, they are working with, and we will get proper `sender` values according to modules/apps in history records.